ssl/ssl_cipher.cc

```diff
@@ -627,6 +651,26 @@ bool ssl_cipher_get_evp_aead(const EVP_AEAD **out_aead,
     }
 
     *out_mac_secret_len = SHA_DIGEST_LENGTH;
+  } else if (cipher->algorithm_mac == SSL_SHA256) {
+    if (cipher->algorithm_enc == SSL_AES128) {
+      if (version == TLS1_VERSION) {
+        *out_aead = EVP_aead_aes_128_cbc_sha256_tls_implicit_iv();
+        *out_fixed_iv_len = 16;
+      } else {
+        *out_aead = EVP_aead_aes_128_cbc_sha256_tls();
+      }
+    } else if (cipher->algorithm_enc == SSL_AES256) {
```

Comment:
None of the cipher suites in the cipher suite list specify `CBC` + `aes256` + `sha256`. So, this looks like dead code.